orchestrator/services/coordinator_service.py

```
@@ -603,6 +611,70 @@ async def _inject_task_output_into_field(
+        save tokens and a ``RUN_FIELD_CONTEXT_DROPPED`` warning is emitted — the
+        budget-gate checkpoint for field context.
+        """
+        field = self._get_field()
+        if not field:
+            return
+
+        # Budget-gate: drop the digest rather than spend tokens we don't have.
+        try:
+            from modules.coordination.dispatcher import BudgetStatus, MissionDispatcher
+            status = MissionDispatcher._get_budget_status(run)
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)